simapp/simd/cmd/root.go

```diff
@@ -83,6 +83,12 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 	a := appCreator{encodingConfig}
 	server.AddCommands(rootCmd, simapp.DefaultNodeHome, a.newApp, a.appExport, addModuleInitFlags)
 
+	// set the default command output properly
+	rootCmd.PersistentPreRun = func(cmd *cobra.Command, args []string) {
+		cmd.SetOut(cmd.OutOrStdout())
```

Comment:
Doing a search of "cmd.SetOut(cmd.OutOrStdout())", we do it in several places. Do you think we can remove all those instances, and just keep this?

Reply:
@amaurym Thanks for the heads up. I've removed all the duplicated calls.